### ISSUE TITLE
Events

### ISSUE THREAD

**Issue by fortmarek** [Vaultio/EtherKit#30]
Events are used to communicate data with "front-end" and are the only way to obtain information from a smart contract, AFAIK.

Do you plan to support them in any way?

Thanks 🙂

Marek from Ackee.

**Comment by SirensOfTitan** [comment]
Absolutely.  I can't say an ETA, but we'll probably add these pretty soon.

**Comment by fortmarek** [comment]
Any status update on this? This is the last blocker for our dev kit, so please keep us informed about any changes 🙂
